### DISCUSSION TITLE
[Script request] AirDC++

### DISCUSSION THREAD

**Discussion by knario** [community-scripts/ProxmoxVE#769] [Request script]
### Application Name

AirDC++

### Website

http://airdcpp.net/

### Description

AirDC++ Web Client is a locally installed application, which is designed for flexible sharing of files or directories within groups of people in a local network or over internet.

The daemon application can be installed on different types of systems, such as on file servers and NAS devices, while the responsive Web UI scales from mobile phones to desktop PCs.

The file sharing communities scale from a few users to thousands of users, while a single client can be used to share millions of files or more than one petabyte (1000 terabytes) of data.

using the dockerfile from this github, I managed to install it on a debian lxc, Although this is probably an impractical way to

I think it would be nice to have it as a script

[https://github.com/gangefors/docker-airdcpp-webclient](url)

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Metigoth** [comment]
This should help with getting script written it is the dockerfile for the docker container.
https://github.com/gangefors/docker-airdcpp-webclient/blob/master/Dockerfile
